crates/bevy_ui/src/ui_node.rs

```diff
@@ -16,7 +16,15 @@ use std::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Sub, SubAssign};
 #[reflect(Component, Default)]
 pub struct Node {
     /// The size of the node as width and height in pixels
-    pub size: Vec2,
+    ///
+    /// This Field should only read by the user
```

Comment:
```suggestion
    /// The node size, automatically calculated by [...]
```
...and insert the system name/label if public, and the phase during which it's updated. Also remove leading empty comment line `///` above.